src/pages/Home/Home.tsx

```diff
@@ -15,12 +19,74 @@ function setRoutes() {
 
 function Home() {
     setRoutes();
```

Comment:
what is setRoutes? is it needed

Reply:
for now, it's not needed, but we will need it once we start adding more routes to the home page, route example "home/products"